demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -103,13 +104,27 @@ def _does_pack_metadata_exist(self):
 
     def is_bump_required(self):
```

Comment:
can you add a docstring?
It is hard to follow the logic(at least for me). e.g: what this phrase means `if len(master_metadata) > 0:`

Reply:
That's just a check that the metadata exists. If the length is longer than 0, we know it can be parsed.